src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,32 @@
+.container {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 16px;
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+}
+
+.stars > :not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
https://github.com/mate-academy/layout_product-cards#tips--hints
> Reuse stars block from [Stars task](https://github.com/mate-academy/layout_stars) and keep it operational. If you change stars--4 modifier to stars--2 or other - number of selected stars should change correctly.

Looks like this part of code is working, but it is not operational